README.md

```diff
@@ -18,14 +18,20 @@ package to retrieve (almost) any browser's history on (almost) any platform.
 
 ```python
 from browser_history.browsers import Firefox
+from browser_history.utils import *
```

Comment:
`import *` is frowned upon in python (unlike many other languages). The [zen of python](https://www.python.org/dev/peps/pep-0020/) says "Explicit is better than implicit.".
```suggestion
from browser_history.utils import get_history
```